triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -157,14 +185,18 @@ object Server {
       maxInboundMessageSize: Int,
       dar: Option[Dar[(PackageId, Package)]],
   ): Behavior[Message] = Behaviors.setup { ctx =>
+    // Init maps.
+    triggers = Map.empty
```

Comment:
this seems super shady and reinforces my point that this should be a class. Contrary to my point above this does actually seem to get worse in this PR.

Reply:
Fixed "enough" for the moment, see previous conversation.